duckduckgo/Android
# Email and other translations

Task/Issue URL: https://app.asana.com/0/488551667048375/1200642600035026
Tech Design URL: 
CC: 

**Description**:
This PR adds translations for email, PUL, favorites and app links.

**Steps to test this PR**:
1. Make sure the app builds
1. Have a look at the above features to double check translations show up

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
